src/plays/navbar/navListItems.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react'
+import Collasp from '../navbar/assest/img.png'
+import LogoImage from '../navbar/assest/logo.png'
+import img1 from "./assest/Chart.png"
```

Comment:
Can you give a meaningful name to these images?
```suggestion
import ChartImg from "./assest/Chart.png"
```